### ISSUE TITLE
Unnecessary phrase in the Spring REST docs reference

### ISSUE THREAD

**Issue by galmegiz** [spring-projects/spring-framework#30513]
In official [Spring REST docs](https://docs.spring.io/spring-restdocs/docs/3.0.0/reference/htmlsingle/) "Build configuration", there is a block that says 
~~~groovy
        "test {
                  outputs.dir snippetsDir
        }". //Configure the test task to add the snippets directory as an output.
~~~

In [stackoverflow question](https://stackoverflow.com/questions/38057946/spring-rest-doc-not-producing-html) 

Wilkinson stated, "Setting the generated snippets directory as an output of the test task and an input of the Asciidoctor task allows Gradle to figure out task dependencies and perform accurate up-to-date checks."

I understand the intention behind that statement, but many people are interpreting test task block as defining the output directory for generated snippets, which is causing confusion and difficulties for them.(Many people mistakenly believe that they can change the snippet generation path by modifying outputs.dir.)

If the dependencies are properly managed through "depends on," I think the use of `outputs.dir` and  becomes unnecessary in this case.

If that statement is not essential and can lead to confusion, it might be a good idea to remove it from the 'Spring Rest docs' manual. Indeed, since it is not intended to be a comprehensive guide on using Gradle, it may be appropriate to omit detailed instructions related to Gradle configuration from the manual. Or it would be beneficial to provide additional explanatory information to clarify the purpose and usage of `outputs.dir.`

In conclusion, I propose removing the "outputs.dir snippetsDir" phrase from the test task block in the manual.

**Comment by bclozel** [comment]
Please raise this issue against the [Spring REST docs issue tracker](https://github.com/spring-projects/spring-restdocs/issues).
